app/controllers/users.controller.js

```diff
@@ -0,0 +1,20 @@
+const httpStatusCodes = require('http-status-codes');
+const { hashPassword } = require('../services/encryptions');
```

Comment:
I think these kinds of methods could be in a helper, what do you think?

Reply:
yes, It could be